sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -106,6 +131,33 @@ namespace Azure { namespace Core { namespace Http {
      * @brief When `true`, allows an invalid certificate authority.
      */
     bool IgnoreUnknownCertificateAuthority = false;
+
+    /**
+     * Proxy information.
+     */
+
+    /**
+     * @brief If True, enables the use of the system default proxy.
+     *
+     * @remarks Set this to "true" if you would like to use a local HTTP proxy like "fiddler" to
+     * capture and analyze HTTP traffic.
+     *
+     * Set to "false" by default because it is not recommended to use a proxy for production and
+     * Fiddler's proxy interferes with the HTTP functional tests.
+     */
+    bool EnableSystemDefaultProxy = false;
+    /**
+     * @brief Proxy information.
+     *
+     * @remark The Proxy Information string is composed of one or more
+     * strings formatted as follows:
+     * ([<scheme>=][<scheme>"://"]<server>[":"<port>])
+     *
+     * Each element should be separated with semicolons or whitespace.
+     */
+    std::string ProxyInformation{};
```

Comment:
These need to be documented, and also `{}` is unnecessary here, it will confuse code and doc readers a bit.